.changeset/ten-ladybugs-sin.md

```diff
@@ -0,0 +1,10 @@
+---
+"@pnpm/exportable-manifest": patch
+"@pnpm/plugin-commands-rebuild": patch
+"@pnpm/lockfile-utils": patch
+"@pnpm/fs.find-packages": patch
+"@pnpm/lifecycle": patch
+"pnpm": patch
+---
```

Comment:
Do we think this changelog is helpful or just adds noise? I included the packages that had the nullish coalescing or optional chaining fixes.

Although I'm fairly confident the auto-fixes are correct, there's a very small change they result in behavior changes that some users might care about?